additionalCSS/aboutMe.css

```diff
@@ -66,6 +66,11 @@
   border-radius: 4px;
 }
 
+.get-my-resume-btn:hover {
+  background: #008552;
+  transform: translateY(-20px);
+}
+
 @media screen and (min-width: 768px) {
   .aboutMeBtn {
```

Comment:
- Great Job for sticking to the givenn design, I think it would  be better if you added addition space between the card and about section to have better UI 👌
### Your Design 👇
![2023-05-05_09-19](https://user-images.githubusercontent.com/52010789/236390343-7be9cfa6-6c1a-4fff-a62b-09f8df95a8e5.png)
### Figma Design 👇
![2023-05-05_09-20](https://user-images.githubusercontent.com/52010789/236390349-fb9c5370-5424-42d9-bb74-956e371361d4.png)